app/forms/participant_form.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ParticipantForm
+  include ActiveModel::Model
+
+  attr_accessor :full_name, :email, :mentor_id
+
+  validates :email, format: { with: Devise.email_regexp }
+  validate :email_is_not_in_use
+
+  def email_already_taken?
+    if user_id
+      User.where.not(id: user_id).exists?(email: email)
+    else
+      User.exists?(email: email)
+    end
+  end
```

Comment:
probably! it was copied and pasted from an existing form in this repo and i haven't actually touched or tested this code, just the mentor part

Reply:
It isn't private because there's a separate page for when an email address is already in use: https://github.com/DFE-Digital/early-careers-framework/blob/induction-tutor-manage-participants/app/controllers/admin/schools/induction_coordinators_controller.rb#L43

It looks like this flow is going to follow a similar pattern, so I'll keep it public.